Detectors/TRD/simulation/include/TRDSimulation/Digitizer.h

```diff
@@ -96,12 +102,6 @@ class Digitizer
     kEmbeddingEvent
   };
 
-  // Trigger parameters
-  bool mTriggeredEvent = false;
-  static constexpr double READOUT_TIME = 3000;                  // the time the readout takes, as 30 TB = 3 micro-s.
```

Comment:
Why a double? and more importantly given the discussion about 30 and 60 timebins on friday, this is going to tie it down. Similarly for the next 2 fields, or do we never simulate for Kr ?